nservicebus/messaging/uniformsession.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Uniform Session between message session and pipeline
+summary: Uniform Session allows to bridge the gap between the message session and the pipeline context and enables dependency injection.
```

Comment:
maybe something like:
Uniform Session introduces a uniform message session across the endpoint
?